apps/public/src/app/(map)/locatie/kies/_components/SideBar/SideBar.tsx

```diff
@@ -0,0 +1,50 @@
+import { Column, Heading, Icon, Paragraph, Link } from '@amsterdam/design-system-react'
+import { ChevronLeftIcon } from '@amsterdam/design-system-react-icons'
+import NextLink from 'next/link'
+import { useEffect, useState } from 'react'
+
+import type { Coordinates } from '../../page'
+import { getAddressFromCoordinates } from '../../utils'
+
+type Props = {
+  coordinates: Coordinates | null
+}
+
+export const SideBar = ({ coordinates }: Props) => {
+  const [address, setAddress] = useState<string | null>(null)
+
+  useEffect(() => {
+    const getAddress = async () => {
+      if (!coordinates) return
+      const result = await getAddressFromCoordinates({ lat: coordinates.lat, lon: coordinates.lon })
+
+      setAddress(result.response.docs[0].weergavenaam)
```

Comment:
I think we should handle this in `getAddressFromCoordinates`, so we just get the `weegavenaam` back from it. Or do you think we'll use the rest of that data in the future?

Reply:
You're right. Since it first was solely the reverse geo service I returned the whole object. But it is specifically a getAddress function now. Will change it.